helper/ResponseVariableFormatter.php

```diff
@@ -158,14 +159,12 @@ public static function formatStructuredVariablesToItemState(
 
             $itemResults = [];
             foreach ($itemResult['taoResultServer_models_classes_ResponseVariable'] as $var) {
-                /**
-                 * @var $responseVariable \taoResultServer_models_classes_ResponseVariable
-                 */
+                /** @var $responseVariable ResponseVariable */
                 $responseVariable = $var['var'];
 
                 if ($responseVariable->getBaseType() === 'file') {
                     $itemResults[$responseVariable->getIdentifier()] = [
-                        'response' => ['base' => ['file' => ['uri' => $var['uri']]]]
+                        'response' => ['base' => ['file' => self::formatVariableFile($responseVariable)]]
```

Comment:
Why did you have to solve it this way, by embedding an encoded file instead a `uri`? It seems this PR https://github.com/oat-sa/extension-tao-outcomeui/pull/374 should have made file download possible through a specific endpoint using the `uri` provided

Reply:
That's a good question. It works on the Results page but it's not good on Test Review mode. For some reason, URIs are not supported there. This change solves the issue without regressions on the Results/Results Review features.

Reply:
Hmm, but it looks like a step back, sending whole file contents even if the user might not play the recording. I guess the solution has a fallback, if file data is already supplied then request with `uri` is not performed. Maybe then this could be a quick solution specific for the client, and having another ticket to use the `uri` for download in test review mode as well 🤔